src/applications/personalization/common/helpers.js

```diff
@@ -77,3 +77,24 @@ export function makeUserObject(options = {}) {
     meta: { errors: null },
   };
 }
+
+/**
+ * Formats a full name from the given first, middle, last, and suffix
+ *
+ * @export
+ * @param {*} {
+ *   first = '',
+ *   middle = '',
+ *   last = '',
+ *   suffix = '',
+ * }
+ * @return {string} The full name formatted with spaces
+ */
+export function formatFullName({
+  first = '',
+  middle = '',
+  last = '',
+  suffix = '',
+}) {
+  return [first, middle, last, suffix].filter(name => !!name).join(' ');
+}
```

Comment:
I feel like this should be a platform utility... I see `src/platform/forms-system/src/js/fields/FullNameField.jsx` does this but it wraps it all in a div + strong tag. No need to change anything.